src/Workspaces/Core/Portable/Options/OptionsHelpers.cs

```diff
@@ -36,5 +36,16 @@ public static T GetOption<T>(OptionKey optionKey, Func<OptionKey, object?> getOp
 
             return (T)value!;
         }
+
+        public static object? GetPublicOption(OptionKey optionKey, Func<OptionKey, object?> getOption)
```

Comment:
Should this be called GetOption to match the GetOption<> overload above? Or name the other one? The core code in OptionSet.cs threw me off in that one called "GetPublicOption" and the rest didn't and that seemed like an oversight...

Reply:
Sure, will rename and doc in a follow-up PR.